### PR TITLE
CLOUDP-66847: mongocli atlas clusters onlinearchive describe

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ [CLOUDP-66847](https://jira.mongodb.org/browse/CLOUDP-66847)

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```bash
mongocli atlas clusters onlinearchive describe 5f04b0e13fe0b32bb2d78af8 --clusterName testShard
{
	"_id": "5f04b0e13fe0b32bb2d78af8",
	"clusterName": "testShard",
	"collName": "test",
	"criteria": {
		"dateField": "created_at",
		"expireAfterDays": 2
	},
	"dbName": "test",
	"groupId": "5e429f2e06822c6eac4d59c9",
	"paused": false,
	"state": "PENDING"
}
```

### CODE REVIEW

**Comment by gssbzn** [comment]
Depends on https://github.com/mongodb/go-client-mongodb-atlas/pull/107

**Comment by gssbzn** [comment]
@robcarlan-mlab ready for review

File: internal/cli/atlas/onlinearchive/describe.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	result, err := opts.store.OnlineArchive(opts.ConfigProjectID(), opts.clusterName, opts.archiveID)
+
+	if err != nil {
+		return err
+	}
+
+	return json.PrettyPrint(result)
+}
+
+// mongocli atlas cluster(s) describe <name> --projectId projectId
```
Comment (robcarlan-mlab) [inline internal/cli/atlas/onlinearchive/describe.go:52]: [np] also needs --clusterName here?

**Comment by gssbzn** [comment]
failing tests are flaky known

**Pull request merged**